Maestro/subscriptions.json

```diff
@@ -826,6 +826,7 @@
         "https://github.com/aspnet/EntityFrameworkCore/blob/release/**/*",
         "https://github.com/aspnet/Extensions/blob/master/**/*",
         "https://github.com/aspnet/Extensions/blob/release/**/*",
+        "https://github.com/aspnet/razor.vscode/blob/master/**/*",
```

Comment:
I think this will work, but just to be safe, would be good to match the casing of the repo.

![image](https://user-images.githubusercontent.com/2696087/53039605-650ff880-3434-11e9-925f-75ecefffdd4d.png)

Reply:
It will, but agree for cleanliness:

https://github.com/dotnet/versions/blob/82ad5fa276bcec84548ba7e31d747be0164f99f5/Maestro/src/Microsoft.DotNet.Maestro.WebApi/Models/SubscriptionsModel.cs#L83-L92

Could consider including the release pattern trigger. Might as well, if there are ever going to be release branches building in the repo. (There doesn't need to be something already there.)

Reply:
added the release/* pattern.